server/sonar-server/src/main/java/org/sonar/server/computation/issue/FinalIssues.java

```diff
@@ -37,4 +38,7 @@ public FinalIssues(TempFolder tempFolder, System2 system2) throws IOException {
     super(tempFolder.newFile("issues", ".dat"), system2);
   }
 
+  FinalIssues(File file, System2 system2) {
```

Comment:
not convinced by the FinalIssues class name. Could IssuesCache be a better name ?